examples/absorption_heatpumps_and_chiller/cooling_cap_dependence_on_cooling_water_temp.py

```diff
@@ -0,0 +1,86 @@
+
+import oemof.thermal.absorption_heatpumps_and_chillers as abs_hp_chiller
+import matplotlib.pyplot as plt
+import os
+import pandas as pd
+
+filename = os.path.join(os.path.dirname(__file__),
+                        'data/characteristic_parameters.csv')
+charpara = pd.read_csv(filename)
+chiller_name = 'Kuehn'
+
+t_cooling = [23, 25, 27, 29, 31, 33, 35, 36, 37, 38, 39, 40]
+
+ddt_75 = abs_hp_chiller.calc_characteristic_temp(
+    t_hot=[75],
+    t_cool=t_cooling,
+    t_chill=[15],
+    coef_a=charpara[(charpara['name'] == chiller_name)]['a'].values[0],
+    coef_e=charpara[(charpara['name'] == chiller_name)]['e'].values[0],
+    method='kuehn_and_ziegler')
+Q_dots_evap_75 = abs_hp_chiller.calc_heat_flux(
+    ddts=ddt_75,
+    coef_s=charpara[(charpara['name'] == chiller_name)]['s_E'].values[0],
+    coef_r=charpara[(charpara['name'] == chiller_name)]['r_E'].values[0],
+    method='kuehn_and_ziegler')
+Q_dots_gen_75 = abs_hp_chiller.calc_heat_flux(
+    ddts=ddt_75,
+    coef_s=charpara[(charpara['name'] == chiller_name)]['s_G'].values[0],
+    coef_r=charpara[(charpara['name'] == chiller_name)]['r_G'].values[0],
+    method='kuehn_and_ziegler')
+COPs_75 = [Qevap / Qgen for Qgen, Qevap in zip(Q_dots_gen_75, Q_dots_evap_75)]
+
+ddt_80 = abs_hp_chiller.calc_characteristic_temp(
+    t_hot=[80],
+    t_cool=t_cooling,
+    t_chill=[15],
+    coef_a=charpara[(charpara['name'] == chiller_name)]['a'].values[0],
+    coef_e=charpara[(charpara['name'] == chiller_name)]['e'].values[0],
+    method='kuehn_and_ziegler')
+Q_dots_evap_80 = abs_hp_chiller.calc_heat_flux(
+    ddts=ddt_80,
+    coef_s=charpara[(charpara['name'] == chiller_name)]['s_E'].values[0],
+    coef_r=charpara[(charpara['name'] == chiller_name)]['r_E'].values[0],
+    method='kuehn_and_ziegler')
+
+
+fig1 = plt.figure()
```

Comment:
```suggestion
fig1 = plt.figure(figsize=(8, 6))
```
If plt.show() is not commented out the figure is cut off in parts. Setting figsize solves this problem.